### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by BipinKalra**

### CODE REVIEW

**Comment by vdvibhu20** [comment]
there is too much padding-x on phone screen

**Comment by BipinKalra** [comment]
> there is too much padding-x on phone screen

phone screen mein i know. voh kam is PR mein kia nhi hai. while doing phone designs, yeh vali cheez us branch mein fix krdi hai maine.

File: examples/views/code-window.hbs [hunk 1]
```diff
@@ -145,7 +145,7 @@
             <div class="pt-4 pb-4 px-xl-5 px-4 input-div">
               <div class="row justify-content-between align-items-center">
                 <div class="col-4 white extra-bold pr-0">
-                  <i class="fas fa-angle-down mr-xl-3 mr-2"></i>
+                  <i class="fas fa-angle-up mr-xl-3 mr-2"></i>
```
Comment (vdvibhu20) [inline examples/views/code-window.hbs]: Screenshot 2019-08-06 at 1 58 38 PM
right align max-points and difficulty

File: examples/views/competition.hbs [hunk 2]
```diff
@@ -1,225 +1,226 @@
 <div class="a-hb">
-  {{>nav-hb}}
-  {{>sidenav}}
+  <div>
```
Comment (vdvibhu20) [inline examples/views/competition.hbs]: Screenshot 2019-08-06 at 2 00 41 PM

**Pull request merged**